### PR TITLE
Upload emotes

### PR DESCRIPTION
**Pull request by TroyDota**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -37,21 +59,3 @@ require (
...
-
-require (
-	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
-	github.com/fsnotify/fsnotify v1.5.1 // indirect
-	github.com/go-redis/redis/v8 v8.11.4 // indirect
-	github.com/google/go-querystring v1.1.0
-)
-
-replace github.com/SevenTV/Common => ./Common
```
Comment (AnatoleAM) [inline go.mod:57]: FeelsDankMan

File: src/server/v3/emotes/emotes.create.go [hunk 2]
```diff
@@ -256,7 +257,21 @@ func create(gCtx global.Context, router fiber.Router) {
...
 
-			// At this point we can add the emote to the database @Anatole DinkDonk DinkDonk
+			// Create the emote in DB
+			eb := structures.NewEmoteBuilder(&structures.Emote{
+				ID:       id,
+				OwnerID:  actor.ID,
+				Name:     name,
+				Status:   structures.EmoteStatusPending,
+				Tags:     tags,
+				Sizes:    []structures.EmoteSize{},
```
Comment (TroyDota) [inline src/server/v3/emotes/emotes.create.go:267]: Perhaps store the original information about the uploaded file, SUCH as the format, width, height, byte size, frame count.

File: src/server/v3/emotes/emotes.process.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+func (epl *EmoteProcessingListener) Listen() {
+	rmq := epl.Ctx.Inst().Rmq
+	if rmq == nil { // RMQ not set up; ignore
+		return
+	}
+
+	// Update queue
+	queueName := epl.Ctx.Config().Rmq.UpdateQueueName
```
Comment (TroyDota) [inline src/server/v3/emotes/emotes.process.go:31]: There is another queue, called result queue, which contains the results

Comment (AnatoleAM) [reply src/server/v3/emotes/emotes.process.go:31]: That's what I was looking for 👍

**Review by AnatoleAM — APPROVED** [review]

**Pull request merged**